### PR TITLE
docs: improve French doc

### PR DESCRIPTION
**Pull request by adelpeuch**
##### Checklist

-   [x] the [individual contributor license agreement][] is signed
-   [x] commit message follows [commit guidelines][]
-   [ ] tests are included
-   [x] documentation is changed or added
-   [ ] [message properties][] have been updated with new phrases
-   [ ] view conforms with [WCAG 2.0 AA][]

##### Description of change

[individual contributor license agreement]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#individual-contributor-license-agreement
[commit guidelines]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#commit
[message properties]: https://github.com/Jasig/uPortal/tree/master/uportal-war/src/main/resources/properties/i18n
[WCAG 2.0 AA]: https://www.w3.org/WAI/WCAG20/quickref/?levels=aaa&technologies=smil%2Cpdf%2Cflash%2Csl

### CODE REVIEW

File: docs/fr/HISTORIQUE.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+* Requires 1.6.0_26 or later
+
+### Spécifications de Servlet Requises
+
+|     | 3.1 | 3.0 | 2.5 | 2.4 | Tomcat Version |
+| --- | --- | --- | --- | --- | -------------- |
+| 5.x |  Y  |  N  |  N  |  N  | 8 or 8.5       |
+| 4.3 |  Y  |  N  |  N  |  N  | 8              |
+| 4.2 |  ?  |  Y  |  N  |  N  | 7 (maybe 8)    |
```
Comment (ChristianMurphy) [inline docs/fr/HISTORIQUE.md]: :question: "maybe" -> "peut-être"?

File: docs/fr/HISTORIQUE.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+| 4.0 |       Y       |       Y       |
+| 3.2 |       N       |       Y       |
+| 3.1 |       N       |       Y       |
+| 3.0 |       N       |       Y       |
+
+## Version de uPortal depuis la 4.0
+
+### 5.2
+
+  - [5.2.1 06-Aug-2018](https://github.com/Jasig/uPortal/releases/tag/v5.2.1)
```
Comment (ChristianMurphy) [inline docs/fr/HISTORIQUE.md]: :question: "Aug' -> "août"?

Comment (adelpeuch) [reply docs/fr/HISTORIQUE.md]: De Fait - correction en cours

Comment (adelpeuch) [reply docs/fr/HISTORIQUE.md]: Done

File: docs/fr/HISTORIQUE.md (hunk 1 above)
Comment (adelpeuch) [reply docs/fr/HISTORIQUE.md]: Done

File: docs/fr/HISTORIQUE.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+uPortal est un projet mature en continuel développement depuis 1999. Cette page fournie une information générale sur les précédentes versions de uPortal.
+
+## Tables de compatibilité
+
+### Java: Version(s) Supportées
+
+|     |  9  |  8  |  7  |  6  |  5  |
+| --- | --- | --- | --- | --- | --- |
+| 5.x |  ?  |  Y  |  N  |  N  |  N  |
```
[the remark this answers is not in the document]
Comment (adelpeuch) [reply docs/fr/HISTORIQUE.md]: Done

File: docs/fr/HISTORIQUE.md (hunk 2 above)
Comment (adelpeuch) [reply docs/fr/HISTORIQUE.md]: Sorry for multiple commits, it was my first time.

**Pull request merged**